apps/angular/react-in-angular/README.md

```diff
@@ -0,0 +1,13 @@
+# React in angular
+
+> author: wandrille-guesdon
```

Comment:
you need to create a file inside the docs/src/content/authors otherwise the application will not build

Reply:
Thanks, ti's done